### PR TITLE
Test and refactoring branch

### PR DESCRIPTION
**Pull request by cupatea**
Add basic integration test for admin

### CODE REVIEW

File: test/fixtures/users.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+vlad:
+  name: Vladyslav Example
+  email: vdavyden@example.com
+  encrypted_password: <%= Devise::Encryptor.digest(User, 'password') %>
```
Comment (verkligheten) [inline test/fixtures/users.yml]: при создании юзера девайс сам сделает encrypted_password
можно просто password: 'password'

File: Gemfile [hunk 2]
```diff
@@ -14,6 +14,7 @@ gem 'devise'
 gem 'carrierwave', '>= 1.0.0.rc', '< 2.0'
 gem "mini_magick"
 gem 'active_interaction', '~> 3.4'
+gem 'rails-controller-testing'
```
Comment (verkligheten) [inline Gemfile]: wtf?

Comment (cupatea) [inline Gemfile]: Error:
AdminUsersIndexTest#test_access_to_users_index_provided_for_admin:
NoMethodError: assert_template has been extracted to a gem. To continue using it,
        add `gem 'rails-controller-testing'` to your Gemfile.
    test/integration/admin_users_index_test.rb:13:in `block in <class:AdminUsersIndexTest>'

File: test/fixtures/users.yml (hunk 1 above)
Comment (cupatea) [inline test/fixtures/users.yml]: Error:
AdminUsersIndexTest#test_access_to_users_index_forbidden_for_non_admin:
ActiveRecord::Fixture::FixtureError: table "users" has no column named "password".

**Pull request merged**